test/test_autograd.py

```diff
@@ -2180,6 +2176,25 @@ def run_test(upper, dims):
             run_test(upper, dims)
             run_test(upper, dims)
 
+        # Testing the inverse using standard inverse and cholesky
+        def mat_test(mat_size, batch_sizes, upper):
```

Comment:
I'm not sure if we want to specifically enforce this. Could we just check that the gradient is always symmetric?

Reply:
I thought this was an interesting test to have, but I can leave it at the symmetry test. Thank you again.